src/constants/index.js

```diff
@@ -0,0 +1,15 @@
+export const CARD_TYPES = {
+  CLOSED: 'closed',
+  FORM: 'form',
+  LARGE: 'lg',
+  SMALL: 'sm',
+  STATUS: 'status',
+  TICKET: 'ticket'
+};
+
+export const STATUS_OPTIONS = {
+  NEW: 'New',
+  IN_PROGRESS: 'In Progress',
+  RESOLVED: 'Resolved',
+  REOPEN: 'Reopen'
```

Comment:
Is this constant defining the different states a ticket can be in?

If so, I'm under the impression there should only be three states. `New`, `In Progress`, and `Closed`. This is shown on page 21 of the Figma doc.

Reply:
Thanks! These have been updated